C/CoinUtils/build_tarballs.jl

```diff
@@ -0,0 +1,82 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "CoinUtils"
+version = v"2.11.4"
+
+# Collection of sources required to complete build
+sources = [
+    # GitSource("https://github.com/coin-or/CoinUtils.git", "d4f2b7f1897b67da6929ab42aa6b1962a388c5b9"),
+    ArchiveSource("https://github.com/coin-or/CoinUtils/archive/releases/2.11.4.tar.gz",
```

Comment:
Would be nice to have this as GitSource (although either ways is fine). In that case the commit is the hash for the release from that repo - f709081c9b57cc2dd32579d804b30689ca789982